parse/parse.go

```diff
@@ -259,9 +260,17 @@ func parseResponseFormat(req *http.Request) string {
 	if IsBrowser(req, true) {
 		return "html"
 	}
+
+	if isYaml(req) {
+		return "yaml"
+	}
 	return "json"
 }
 
+func isYaml(req *http.Request) bool {
+	return req.Header.Get("Accept") == "application/yaml"
```

Comment:
A client can send multiple Accept headers to say different content types they accept.  So you need to see if it's in one of the headers